test/db/formats/mach0/swift

```diff
@@ -18,8 +18,8 @@ C*~Cd?
 Cd~?
 EOF
 EXPECT=<<EOF
-114
-134
+115
```

Comment:
which is the line that is new? can we have a test that provides an insight of the fix instead of just a row count?

Reply:
Previously all lines looked like this:

```
Cd 8 @ 0x100005000
Cd 8 @ 0x100005010
Cd 8 @ 0x100005018
Cd 8 @ 0x100005020
```

Now there's also this line:

```
CCu base64:U3dpZnQuU3RyaW5nLmluaXQoX2J1aWx0aW5TdHJpbmdMaXRlcmFsOiBCdWlsdGluLlJhd1BvaW50ZXIsIGJ5dGVTaXplOiBCdWlsdGluLldvcmQsIGlzQVNDSUk6IEJ1aWx0aW4uSW50MSkgLT4gU3dpZnQuU3RyaW5n @ 0x10000440a
```

So in hindsight we should probably instead make the "grep" pattern stricter so it doesn't result in false positives when `Cd` just happens to be in an unrelated line's base64-encoded data.

Reply:
Fixed it by adding `^` to make this test less brittle.